packages/ckeditor5-ckbox/src/utils.ts

```diff
@@ -135,7 +135,7 @@ export function sendHttpRequest( {
 	authorization
 }: {
 	url: URL;
-	signal: AbortSignal;
+	signal?: AbortSignal;
```

Comment:
It may make someone forget to handle cancellation of requests in the future. I'd be better if you were specific on call-site that you considered aborting but didn't implement it. Like:
```ts
const response = await sendHttpRequest( {
		url,
		authorization: value,
		signal: ( new AbortController() ).signal // Aborting is unnecessary.
} );
```
and kept this option non-opional here.